### PR TITLE
Support block parameter

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/api-server/src/methods/modules/eth.ts [hunk 1]
```diff
@@ -39,6 +39,9 @@ const SUDT_PAY_FEE_LOG_FLAG = "0x1";
 const POLYJUICE_SYSTEM_LOG_FLAG = "0x2";
 const POLYJUICE_USER_LOG_FLAG = "0x3";
 
+const HEADER_NOT_FOUND_ERR_CODE = -32000;
```
Comment (RetricSu) [inline packages/api-server/src/methods/modules/eth.ts:42]: suggest moving to `packages/api-server/src/methods/error-code.ts`

Comment (classicalliu) [reply packages/api-server/src/methods/modules/eth.ts]: updated

File: packages/api-server/src/methods/error-code.ts [hunk 2]
```diff
@@ -13,3 +13,5 @@ export const METHOD_NOT_SUPPORT = -32000;
 export const WEB3_ERROR = -32001;
 
 export const GW_RPC_REQUEST_ERROR = -32002;
+
+export const HEADER_NOT_FOUND_ERROR = -32000;
```
Comment (RetricSu) [inline packages/api-server/src/methods/error-code.ts:17]:
```suggestion
export const HEADER_NOT_FOUND_ERROR = -32003;
```
-32000 is already in used.

**Review by RetricSu — CHANGES_REQUESTED** [review]

File: packages/api-server/src/methods/error-code.ts (hunk 2 above)
Comment (classicalliu) [reply packages/api-server/src/methods/error-code.ts:17]:
```json
{
    "jsonrpc": "2.0",
    "id": 2,
    "error": {
        "code": -32000,
        "message": "header not found"
    }
}
```

`-32000` is equal to infura response

Comment (RetricSu) [reply packages/api-server/src/methods/error-code.ts:17]: in that sense we should change the other 32000 error to something else.

**Review by RetricSu — APPROVED** [review]

**Pull request merged**